augly/tests/image_tests/functional_unit_tests.py

```diff
@@ -17,6 +17,9 @@ def test_blur(self):
 
     def test_brightness(self):
         self.evaluate_function(imaugs.brightness)
+    
+    def test_skew(self):
```

Comment:
Have you run this test yet? I don't think so since you haven't added a test output file to the `assets` folder. As I've already said, you need to run `skew()` with the default args, then put the output image file in `augly/assets/image/dfdc_expected_output/test_skew.png`. Then run the 2 new tests & make sure they pass.

Reply:
Okay @zpapakipos I'll run and add in the `assets` folder.

Thanks